src/common/ui-components/Map/Map.tsx

```diff
@@ -8,8 +8,7 @@ import MapGL, {
   FlyToInterpolator,
   TransitionInterpolator,
   EasingFunction,
-  TRANSITION_EVENTS,
-  ViewportProps
+  TRANSITION_EVENTS
```

Comment:
this will not work with the current Map setup

Reply:
It's an unused import.

Reply:
oh, true, I didn`t see the deletion in the bottom